README.md

```diff
@@ -51,14 +51,11 @@ We organize webpack CLI as a multi-package repository using [lerna](https://gith
 Supporting developers is an important task for webpack CLI. Thus, webpack CLI provides different commands for many common tasks.
 
 -   [`webpack-cli init`](./packages/init/README.md#webpack-cli-init) - Create a new webpack configuration.
--   [`webpack-cli add`](#) - Add new properties to a webpack configuration file.
```

Comment:
@evenstensberg Could have added a disclaimer about depreciation or to use the previous version for that?